shared/js/background/message-handlers.js

```diff
@@ -58,8 +59,8 @@ export async function setLists (options) {
     }
 
     try {
-        notifyPopup({ closePopup: true })
-        reloadCurrentTab()
+        postPopupMessage({ messageType: 'closePopup' })
+        await reloadCurrentTab()
```

Comment:
In the `reloadCurrentTab` implementation we're not awaiting the [`browser.tabs.reload`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/reload) call. Do we want to wait here for the tab to complete reload, or just that the reload has started?

Reply:
Good catch, I'll fix that now :+1: .